packages/cli/package.json

```diff
@@ -32,7 +33,8 @@
     "prepublish": "rm -rf bin && git checkout bin"
   },
   "dependencies": {
```

Comment:
Can you re-run `pnpm i` to update the `pnpm-lock.yaml` file with the new dependencies?